CONTRIBUTING.md

```diff
@@ -0,0 +1,179 @@
+# Contribution Guidelines
+
+If you are interested in contributing to Substrate Documentation, please follow these contribution 
+guidelines.
+
+* [PR Checklist](#pr-checklist)
+* [Documentation Standards](#documentation-standards)
+* [Documentation Style](#documentation-style)
+
+## PR Checklist
+
+- [ ] Are the audience and objective of the document clear? E.g. a document for developers that 
+  should teach them about transaction fees.
+- [ ] Is the writing:
+  - **Clear**: No jargon
+  - **Precise**: No ambiguous meanings
+  - **Concise**: Free of superfluous detail
+- [ ] Does it follow our [style guide](#documentation-style)?
+- [ ] If this is a new page, does the PR include the appropriate infrastructure, e.g. adding the
```

Comment:
> this is a copy from a different repo, right? 

A greatly reduced copy, but yes. Will leave the bullet point for now since forgetting this is a common mistake, but I'll come back to it whenever we know what the structure is like.